src/components/nothing-here/nothing-here.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { HeadingLevel } from 'src/common/enums';
+import { Heading } from '..';
+import useStyles from './styles';
+
+type NothingHereProps = {
+  children: string;
+};
+
+const NothingHere: React.FC<NothingHereProps> = ({ children: text }) => {
```

Comment:
Maybe it's better to name this component like EmptyMessage?

Reply:
EmptyListMessage